build.gradle.kts

```diff
@@ -196,16 +196,6 @@ task<Wrapper>("wrapper") {
 }
 
 /**
- * Workaround fix for calling [org.gradle.testing.jacoco.plugins.JacocoPluginExtension.applyTo]
- *
- * [Issue details here](https://github.com/gradle/kotlin-dsl/issues/458)
- */
-fun org.gradle.testing.jacoco.plugins.JacocoPluginExtension.applyToHelper(task : Task) {
```

Comment:
In hindsight this was simply because I was passing the wrong type to this method. There was an actual bug in the Gradle API that has been fixed, but I conflated the two issues.